CHANGELOG.md

```diff
@@ -43,6 +44,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Fixed errors in adjoint-only code preventing successful adjoint build
 - Fixed zero convective precipitation and high cloud base in runs using GEOS-FP (>=01Jun2020) or GEOS-IT
 - Updated GEOS-only code for compatibility with GEOS-Chem 14.5
+- Fixed typos in `HEMCO_Config.rc` for CH4 simulations causing mobile combustion emissions to be double counted
+- Fixed handling of FIRST flag in carbon_gases_mod.F to limit log prints to first timestep only
+- Removed extraneous pressure correction in GCHP carbon simulations by adding 'activate: true' to geoschem_config.yml
+- Fixed bug in GC-Classic OCS emissions where unit conversion of km2 to m2 occurred twice
+- Changed dimension of EmisOCS_Total from 2D to 3D since all emissions for all sectors are 2D
```

Comment:
Should this be 3D to 2D?

Reply:
Yes, according to commit https://github.com/geoschem/geos-chem/pull/2554/commits/77d7fea5e0c340dec417f3ce26c103e0af239ba1 it should be 3D to 2D. Now fixed in https://github.com/geoschem/geos-chem/pull/2554/commits/efedff3fee4eedc69b0163cecfa77c9093f23d31.